scripts/globals/job_utils/geomancer.lua

```diff
@@ -425,7 +425,7 @@ xi.job_utils.geomancer.spawnLuopan = function(player, target, spell)
     xi.job_utils.geomancer.addAura(luopan, 0, effect, finalPotency, targetType)
 
     -- Save the mp cost for use with Full Circle on the luopan
-    luopan:setLocalVar("MP_COST", spell:getMPCost())
```

Comment:
```lua
xi.job_utils.geomancer.fullCircle = function(player, target, ability)
    local hppRemaining = target:getHPP()
    local mpCost       = target:getLocalVar("MP_COST")
```
seems the author believed the _target_ of the ability would be the loupan, not the player. could you change that line to say _player_ as well? just for clarity

Reply:
all set, sir

Reply:
ty